great_expectations/rule_based_profiler/rule_based_profiler.py

```diff
@@ -404,6 +393,30 @@ def reconcile_profiler_variables(
 
         return effective_variables
 
+    def reconcile_profiler_variables_as_dict(
```

Comment:
Since we can use these in usage stats, making it public and refactoring primary reconciliation methods.